BackEnd/src/main/java/com/unosquare/admin_core/back_end/configuration/mappings/domain/EventMappings.java

```diff
@@ -17,6 +17,7 @@ protected void configure() {
                 map().setEventType(new EventType(source.getEventTypeId()));
                 map().setStartDate(source.getStartDate());
                 map().setEndDate(source.getEndDate());
+                map().setMessage(source.getMessage());
```

Comment:
Is this required? The mapps are more to do with different naming etc. Convention based mapping should apply.

Reply:
Removed